infrastructure/src/main/kotlin/com/dobby/external/prompt/ExperimentPostKeywordMapper.kt

```
@@ -0,0 +1,84 @@
+                ApplyMethodKeyword(
+                    content = applyMethodDto.content ?: "",
+                    isFormUrl = applyMethodDto.isFormUrl,
+                    formUrl = applyMethodDto.formUrl ?: "",
+                    isPhoneNum = applyMethodDto.isPhoneNum,
+                    phoneNum = applyMethodDto.phoneNum ?: ""
+                )
+            },
+            matchType = dto.matchType?.let { matchTypeStr ->
+                try {
+                    MatchType.valueOf(matchTypeStr)
+                } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)